package/src/components/Channel/Channel.tsx

```diff
@@ -1330,7 +1332,7 @@ const ChannelWithContext = <
     } as StreamMessage<At, Me, Us>;
 
     try {
-      if (!StreamCache.getInstance().currentNetworkState) {
+      if (!isConnected) {
         console.log(t('Something went wrong'));
```

Comment:
this console.log can be reviewed

Reply:
Yeah, don't think we need to translate it necessarily even since it's logged to the console, and not displayed to the end user. I'll change it up.

Reply:
```suggestion
        console.log(`Could not send message: Network is disconnected.`);
```

Something like this, maybe?

Reply:
Sounds good!!